decentraland/unity-renderer
# feat: friend request cancel

## What does this PR change?

Displays a popup with the detailed information when clicking a sent friend request from the request list.

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/friend-request-cancel
2. Open the friend request list
3. Click over a sent friend request
4. A popup should be displayed with the correct information
5. Click on cancel button
6. The request should be cancelled with the chance of having an error

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
